sway/desktop/output.c

```diff
@@ -578,6 +578,28 @@ static void update_textures(struct sway_container *con, void *data) {
 	container_update_marks_textures(con);
 }
 
+static void update_output_manager_config(struct sway_server *server) {
+	struct wlr_output_configuration_v1 *config =
+		wlr_output_configuration_v1_create();
+
+	struct sway_output *output;
+	wl_list_for_each(output, &root->all_outputs, link) {
+		if (output == root->noop_output) {
+			continue;
+		}
+		struct wlr_output_configuration_head_v1 *config_head =
+			wlr_output_configuration_head_v1_create(config, output->wlr_output);
+		struct wlr_box *output_box = wlr_output_layout_get_box(
+			root->output_layout, output->wlr_output);
+		if (output_box) {
+			config_head->state.x = output_box->x;
+			config_head->state.y = output_box->y;
+		}
```

Comment:
We also need to handle DPMS here. We don't want to mark the output as disabled in the protocol if it's just DPMS'ed off.

The easiest way is probably to check whether the `sway_output` is enabled (does it have an `enabled` field?) and overwrite `config_head->enabled`.